tsl/src/compression/compress_utils.c

```diff
@@ -436,6 +437,80 @@ decompress_remote_chunk(FunctionCallInfo fcinfo, const Chunk *chunk, bool if_com
 	return success;
 }
 
+/*
+ * Create a new compressed chunk using existing table with compressed data.
+ *
```

Comment:
Sounds like "attach", not "create"

Reply:
Well, it is not actually attach, since chunk not exists yet, basically no meta-data about it. Additionally this and further API changes are mimicking what we already had for non-compressed chunks

Reply:
As far as I understand, there are two parts of creating a compressed chunk: creating the compressed chunk table and creating the metadata. If you already have the table and want to create the respective metadata, I'd call this operation "attaching a compressed chunk". Do we have other kind of "attach"?

Reply:
It is basically the same as if you `select create_hypertable(table)` when table has data, it still reads as create not attach, right? I guess this is a matter of perspective

Reply:
Yep, but it creates different tables and moves data there. If it was just modifying metadata, I'd also think that it is an "attach" operation. Anyway, don't want to slow you down too much with these terminological discussions :)